src/cmeanvar_gaussian.jl

```diff
@@ -53,8 +51,8 @@ function mean_var(p::CMeanVarGaussian{T,ScalarVar}, z::AbstractArray) where T
         error("Mapping should return eltype $T. Found: $(eltype(ex))")
     end
 
-    μ = ex[1:end-1,:]
-    σ = ex[end:end,:] .* fill!(similar(μ, 1, size(μ,2)), 1)
+    μ, σ = last_split(ex)
+    σ = σ # .* fill!(similar(μ, 1, size(μ)[2:end]...), 1) # everything seems to work fine without this...
```

Comment:
This seems to be unnecessary.

Reply:
did you test it on gpu? the `fill!` was necessary to make it work there. maybe that changed with the new flux/zygote/cuarrays though...

Reply:
Yes, this seems to be working even on gpu now.